tests/test_home_page.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from utilities.config_reader import ReadConfig
+from page_objects.login_page import LoginPage
+from page_objects.home_page import HomePage
+
+
+@pytest.mark.smoke
+def test_suggested_pages_count(initialize_driver):
+    email = ReadConfig.get_user_email()
+    password = ReadConfig.get_user_password()
+    driver = initialize_driver
+    suggested_pages_url = ReadConfig.get_suggested_pages_url()
+    (LoginPage(driver).set_email(email).set_password(password).
+     click_login_button().click_save_info_not_now_button().
+     click_notification_not_now_button().click_see_all_button().
+     wait_url_to_be(suggested_pages_url))
```

Comment:
what is this? 
lots of code duplications all around test files.  Please use fixtures to avoid it. Redo all the tests with fixtures

Reply:
@bohdandovbsyh done. Thanks.